src/components/LookupNotification.tsx

```diff
@@ -348,6 +349,12 @@ async function fetchLookup(
     }
   }
 
+  // check for userToken in the query string
+  const qs = querystring.parse(location.search)
+  if (qs.userToken) {
+    headers['X-OneBlink-User-Token'] = qs.userToken as unknown as string
+  }
```

Comment:
Ideally this comes from the authService as well like the Id token. That way devs can use it without having to know what query string parameters are required to use it.

Reply:
yeah makes sense but auth service comes from apps which has not been updated yet